### PR TITLE
[NUI] Fix recursion when the view sets KeyboardNavigationSupport to true

### PR DESCRIPTION
**Pull request by rabbitfor**
In Dali code, the method GetNextKeyboardFocuableActor should return
an empty actor (= the base handle without a referenced object)
if there is no actor can be focused.

The corresponding method in NUI is GetNextFocusableView.
It returned new View() (that is not an empty actor) which leads to the wrong result.
So instead returning new View, this patch returns a null IntPtr.

Signed-off-by: Jiyun Yang <ji.yang@samsung.com>

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseComponents/CustomView.cs [hunk 1]
```diff
@@ -434,7 +434,7 @@ public virtual void OnFocusLost()
         /// <since_tizen> 3 </since_tizen>
         public virtual View GetNextFocusableView(View currentFocusedView, View.FocusDirection direction, bool loopEnabled)
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/BaseComponents/CustomView.cs:435]: Do not you need to modify the description of the return value in the comment?
empty handle -> null

**Comment by rabbitfor** [comment]
Please note that, the related dali-csharp-binder patch should be merged first. 
(https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/245938/)

File: src/Tizen.NUI/src/public/BaseComponents/CustomView.cs (hunk 1 above)
Comment (rabbitfor) [reply src/Tizen.NUI/src/public/BaseComponents/CustomView.cs:435]: Done!

**Review by huiyueun — APPROVED** [review]

**Review by hyunjushin — APPROVED** [review]

**Comment by rabbitfor** [comment]
I will merge this patch after the intergration of dali-csharp-binder is done.

**Review by hyunjushin — DISMISSED** [review]
Please re-push this patch into DevelNUI branch. DevelNUI will be integrated with master branch on every week.

**Pull request merged**